ptf/sailag.py

```diff
@@ -87,34 +85,25 @@ def createRemoveLagMemberTest(self):
         # verify LAG and LAG members
         attr_list = sai_thrift_get_lag_attribute(
             self.client, lag3, port_list=portlist)
-        # TODO: lag_members = attr_list["SAI_LAG_ATTR_PORT_LIST"].idlist
+        lag_members = attr_list["SAI_LAG_ATTR_PORT_LIST"].idlist
+        print(lag_members)
         count = attr_list["SAI_LAG_ATTR_PORT_LIST"].count
         assert count == 0
 
-        # verify LAG and LAG members
-        attr_list = sai_thrift_get_lag_attribute(
-            self.client, lag3, port_list=portlist)
-        # TODO: lag_members = attr_list["SAI_LAG_ATTR_PORT_LIST"].idlist
-        count = attr_list["SAI_LAG_ATTR_PORT_LIST"].count
-        assert count == 0
```

Comment:
Why is this removed?

Reply:
This is a duplicate of the fragment from lines 87-92 in the original file.